great_expectations/datasource/fluent/file_path_data_asset.py

```diff
@@ -195,6 +195,7 @@ def get_batch_list_from_batch_request(
                     exclude=self._EXCLUDE_FROM_READER_OPTIONS,
                     exclude_unset=True,
                     by_alias=True,
+                    config_provider=self._datasource._config_provider,
```

Comment:
Same note - can we avoid private attr access?

Reply:
I think both of these belong as private methods.